Hello.java

```diff
@@ -0,0 +1,13 @@
+public class Hello {
+     public Hello() {
+         sayHello();
```

Comment:
The role of the constructor is to construct the object for subsequent use as allowed by its methods. Here the constructor does all the useful work. From the client (main) perspective, after constructing the object there is nothing to be done with it.